application/src/bll/redirect.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { PATH } from '../pages/PageRoutes/PageRoutes';
+import { store } from './store';
+
+export const redirect = () => {
+  let passwordLocal;
+  let emailLocal;
```

Comment:
Please don't use `local`, `remote`, etc. as a prefix in the names - it will be necessary to refactor a lot of code/names in case we will swith or implementation to the real backend

Please use just `password` and `email`